adhocracy4/comments_async/templatetags/react_comments_async_with_categories.py

```diff
@@ -14,7 +14,7 @@
 
 
 @register.simple_tag(takes_context=True)
-def react_comments_async_with_categories(context, obj):
+def react_comments_async_with_categories(context, obj, debate_app=False):
```

Comment:
Yes, that's perfect! You can already rename it to react_comments_async, because where we currently use it, we need to change it to say "True" in the end anyway.

Reply:
Actually, it should work and then not break anything, if you rename this and then make a new one like this:
`
@register.simple_tag(takes_context=True)
def react_comments_async_with_categories(context, obj):
    react_comments_async(context, obj, debate_app=True)
`